### PR TITLE
Update for 0.2.0 release

### PR DESCRIPTION
**Pull request by SuperQ**
Update build
* Add `.gitignore`.
* Update Makefile.COMMON from prometheus/utils.

Release 0.2.0
* Update Makefile.
* Add CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+* [CHANGE] Migrate logging to prometheus/log. (#14)
+* [CHANGE] Document use-case without any StatsD server. (#16)
+* [CHANGE] New Dockerfile using alpine-golang-make-onbuild base image (#17)
+* [IMPROVEMENT] Allow configuration of UDP read buffer (#22)
+* [CHANGE] Move from client_golang/model to common/model (#23)
+* [BUGFIX] allow metrics with dashes when mapping (#24)
+* [IMPROVEMENT] add root endpoint with redirect (#25)
+* [CHANGE] rename bridge to exporter (#26)
+* [BIGFIX] fix documentation for docker udp port publishing (#31)
```
Comment (fabxc) [inline CHANGELOG.md]: `BUGFIX` – but this doesn't affect the program itself, so generally don't include this in changelogs.

**Comment by fabxc** [comment]
We haven't released this in ages. But we shouldn't include things like doc fixes and vendoring updates (they are outdated anyway).

**Comment by SuperQ** [comment]
@fabxc Ok, cleaned up CHANGELOG.

**Comment by grobie** [comment]
:+1:

**Pull request merged**